recipes/tscv/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "tscv" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: db0f7d02f462013832eab786d133e2db0481e25cd6b82617cb53cecb7565ce78
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - scikit-learn >=0.22
+    - pip
+  run:
+    - python >=3.6
+    - scikit-learn >=0.22
```

It looks like numpy is also a direct dependency of `tscv`:
```suggestion
    - scikit-learn >=0.22
    - numpy
```